layouts/partials/hooks/body-end.html

```diff
@@ -7,28 +7,63 @@
 {{/* copy-and-paste helper for codenew shortcode */}}
 {{- if or (.HasShortcode "code_sample") (.HasShortcode "code") (.HasShortcode "codenew") -}}
   {{- if hugo.IsProduction -}}
-    {{- $sweetAlert :=  resources.Get "js/sweetalert-2.1.2.min.js" | minify | fingerprint -}}
-<script async src="{{ $sweetAlert.RelPermalink }}" integrity="{{ $sweetAlert.Data.Integrity }}" crossorigin="anonymous"></script>
+    {{- $toastrCss := resources.Get "css/toastr-2.1.4.min.css" -}}
+    <LINK rel="stylesheet" href="/css/toastr-2.1.4.min.css">
```

Comment:
- I'd put this (the CSS load) in `layouts/partials/hooks/head-end.html` so that the CSS loads early.
- This line isn't using `$toastrCss`; why not?